### PR TITLE
Exports path to `contexts` dir and updates domain for `alumni`, `movieTicket` and `safeChef` context URLs

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: credentials/alumni/credential.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
     "https://www.w3.org/2018/credentials/examples/v1",
-    "https://playground.chapi.io/examples/alumni/context-v1.json"
+    "https://contexts.vcplayground.org/examples/alumni-context-v1.json"
```
Comment (JSAssassin) [inline credentials/alumni/credential.json:5]: NOTE: The contexts will be now served by https://github.com/credential-handler/chapi-playground-contexts-server/pull/1

**Review by gannan08 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]
RSLGTM ... I'm not privy to all the reorganization details happening here -- deferring to other reviewers to sign off on the direction.

File: README.md [hunk 2]
```diff
@@ -43,14 +43,20 @@ Please add or update credentials in the `./credenitals` dir. When adding new
 credential, make sure the directory name is camel-cased. The directory should
 contain a logo image (with `.jpg` or `.png` file extensions) and the credential
 in the `credential.json` file. Additionally, you can also add a file for the
-credential context (for example see - `./credentials/movieTicket`) if you want
-to host the contexts in `https://playground.chapi.io/`.
+credential context in the `./contexts` dir (for example see -
+`./contexts/alumni-context-v1.json`) if you want to host the contexts in
+[`@credential-handler/chapi-playground-contexts-server`](https://github.com/credential-handler/chapi-playground-contexts-server).
```
Comment (davidlehn) [inline README.md:48]:
```suggestion
[`@credential-handler/vc-playground-contexts-server`](https://github.com/credential-handler/vc-playground-contexts-server).
```

**Review by davidlehn — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md]: Fixed [077f2a5](https://github.com/credential-handler/vc-examples/pull/14/commits/077f2a5ccd661b733c7e268c885bd50e51dcbd2b)

**Pull request merged**